### PR TITLE
test(integration): add glob for temp dir, update snapshot

### PR DESCRIPTION
**Pull request by azdanov**
When globbing is added test passes. Need to verify why it doesn't
work without it.

### CODE REVIEW

File: test/__snapshots__/integration.spec.ts.snap [hunk 1]
```diff
@@ -1,98 +1,78 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`integrated should lint files 1`] = `
-"Zoe 1.8.4
+"Zoe 1.9.2
```
Comment (jorgegonzalez) [inline test/__snapshots__/integration.spec.ts.snap]: Will this need to be manually updated after every publish?

What happens if we change the default rules and the error messages below should be different? I don't understand how these snapshots work.

File: test/integration.spec.ts [hunk 2]
```diff
@@ -1,18 +1,19 @@
-import { exec, spawn } from 'child_process';
-import { join } from 'path';
-import { promisify } from 'util';
-import { createTemp, removeTemp } from './utils/codegen';
+import { exec, spawn } from "child_process";
```
Comment (jorgegonzalez) [inline test/integration.spec.ts]: Revert everything back to single quotes

File: test/__snapshots__/integration.spec.ts.snap (hunk 1 above)
Comment (azdanov) [reply test/__snapshots__/integration.spec.ts.snap]: Yes, until a better way of testing is made. This way is just super simple, and offers some safety. I'm thinking of comparing files themselves later on.

File: test/integration.spec.ts (hunk 2 above)
Comment (azdanov) [reply test/integration.spec.ts]: Ah, thanks!

Comment (jorgegonzalez) [reply test/integration.spec.ts]: I'll add lint-staged after this is merged to avoid styling errors in the future.

**Comment by jorgegonzalez** [comment]
@azdanov Would you be interested in joining the project as a maintainer? I'd love to have you as part of the team.

**Comment by azdanov** [comment]
@jorgegonzalez Sure! I like what you're trying to accomplish with this project.

**Comment by azdanov** [comment]
Timeout is too low. This test requires a lot of I/O, probably Travis was having a slow-down.

**Comment by jorgegonzalez** [comment]
Just passed.

**Pull request merged**